### ISSUE TITLE
Using Chinese character for 令

### ISSUE THREAD

**Issue by i-h8-github** [KanjiVG/kanjivg#455]
https://en.wiktionary.org/wiki/%E4%BB%A4

Shouldn't the Japanese character be as displayed at https://en.wiktionary.org/wiki/%E4%BB%A4#Japanese?

**Comment by benkasminbullock** [comment]
Both forms are supplied but the handwritten form is the default one. See

https://kanjivg.tagaini.net/viewer.html?file=04ee4-Insatsu.svg

for the wiktionary version.

The references used to decide which form are given on this page:

https://kanjivg.tagaini.net/ref.html

I assume that the original creator of the image found that the above one
was correct for the handwritten case.

**Comment by i-h8-github** [comment]
You are right. After conferring with a few Japanese people, there is indeed a difference between the handwritten and printed styles of that character (interestingly, one they they had not been consciously aware of before I asked).

Thank you for your attention.
